src/interfaces/pair.ts

```diff
@@ -5,6 +5,7 @@ export interface IPair {
   token0: IToken;
   token1: IToken;
   price: string;
+  fee: string;
```

Comment:
Can we do `swapFee` since we also have `platformFee` and thus `fee` is ambiguous here?

Reply:
Second this, and you probably have to edit the test files to make the CI pass